### ISSUE TITLE
Logos

### ISSUE THREAD

**Issue by simonv3** [opensourcedesign/opensourcedesign.github.io#18]
*EDIT for update*:

Hey all!

Thanks so much for contributing all of your great thoughts, ideas, logos, sketches, etc. This is super awesome, and *exactly* what I hope for this community. It's great to see you all pitch in, and I'm super appreciative of all of the work you've been doing on this.

However, to prevent this from going on and on with new ideas and concepts, and never end up with a logo, I'm going to suggest that we all agree that a week from today Wednesday the 17th (this gives people a weekend to pitch in ideas as well) we close this topic. If by the end of this week we don't have a clear choice of favorite, we can put things to a vote.

I want *everyone's* opinions on *our* logo. Why a particular concept is great, why it fails, etc. I want to make sure that everyone's voice gets a chance to be heard. But I also want a positive conclusion to this conversation.

Let me know if you think this is unfair.

*continued from before*

Some concepts so far:
1. ![mariekevc](https://cloud.githubusercontent.com/assets/6065633/8058252/83ca6f4e-0eb8-11e5-9399-dfea4c805a52.png) @MariekevC 
2. ![jdittrich](https://cloud.githubusercontent.com/assets/3416487/8061486/5504bf3a-0ecb-11e5-9f07-9903dd2b2d8b.png) @jdittrich 
3. ![plastelina](https://cloud.githubusercontent.com/assets/7548189/8062660/c992bf26-0ec8-11e5-8015-2225f1daee10.png) @plastelina 

@plastelina. The logo you referenced is semi-off limits. It's the design open logo, for some more background look here: https://github.com/opensourcedesign/jobs/issues/26.

**Comment by teufelabgott** [comment]
Does it make sense to start the conversation defining goals and establishing some constraints for collaborators before posting concepts? 

Just trying to make reference to an [existing resource](https://github.com/opensourcedesign/Design-Brief/blob/master/DESIGN-BRIEF.md).

**Comment by simonv3** [comment]
@teufelabgott that would be excellent. The conversation was running over from #17 though, I just wanted to keep that thread focused. I'm open to any constraints / etc anyone wants to come up with (but a bit too busy myself to do it).

**Comment by jancborchardt** [comment]
I like the right concept by @jdittrich a lot, definitely something to be built upon. The left one with the pencil is a bit too similar to the »DesignOpen« folks. Also the metaphor of Design = Pencil is getting a bit old, no?

**Comment by jancborchardt** [comment]
Will see if I can come up with some sketches too.

**Comment by GarthDB** [comment]
I think the pencil one from @jdittrich is awesome, and different enough from the [Open Design Foundation](http://opendesign.foundation) logo.

**Comment by GarthDB** [comment]
Just playing with some variations to @jdittrich's design.

![opensrcdesign_01](https://cloud.githubusercontent.com/assets/125516/8068338/e7b5d43a-0eb8-11e5-82bc-b8801ce50c70.png)
![opensrcdesign_02](https://cloud.githubusercontent.com/assets/125516/8068339/e7b8b18c-0eb8-11e5-8c82-2466758177d1.png)

I posted them on [Pixelapse](https://www.pixelapse.com/GarthDB/projects/Open%20Source%20Design) if anyone wants the vector files.

**Comment by una** [comment]
Some thoughts! 

![od-ideas](https://cloud.githubusercontent.com/assets/1693164/8071028/079855a8-0ecc-11e5-88d4-c4980faf71b5.png)

**Comment by GarthDB** [comment]
:ship_it:

**Comment by Xaviju** [comment]
I'm not convinced about the pencil, tried with a brush instead. 

![](http://i.imgur.com/AcpGJXc.png)

**Comment by pfefferle** [comment]
I love the variations of the logo by @jdittrich from @GarthDB and @una but I think I would prefer the brush by @Xaviju

**Comment by una** [comment]
I like the brush too!

**Comment by gillisig** [comment]
Yeah that brush is pretty simple and elegant. Good job @Xaviju !

**Comment by elhombretecla** [comment]
The brush logo by @Xaviju its the best option in my opinion. I like it :)

**Comment by jdittrich** [comment]
Visually, I like the brush logo too; its rather elegant; But: imho it communicates a Design-is (like)-Art message which I don’t find ideal.

**Comment by plastelina** [comment]
It's a very good logo @Xaviju , but I'm not convinced - brush? It feels more like a painter then designer. This doesn't really send a message...but this is just my opinion. (agreed with @jdittrich comment)

**Comment by simonv3** [comment]
My thoughts on the brush: I haven't used a brush in my design work ever. But then, I'm also not a pure designer. I liked the pencil for that reason better - I sketch a lot.

I'd love to see some more iterations of it though @Xaviju. Maybe some color? I really like the bottom right of @una's color concepts.

**Comment by jdittrich** [comment]
@una’s concepts: I like the middle-top one most: Few elements, incorporates both circle outlines in the pencil, works well in b/w or if blurred (though here the brush is better; might improve if the main circle and pencil have a darker color than light blue/bright yellow)

**Comment by pfefferle** [comment]
Photoshop and other Apps still use the "brush" as a tool, so I think it is not that far away, from what we do nowadays.

**Comment by jancborchardt** [comment]
As also said above regarding the pen or pencil: I don’t think a brush or these tools properly convey what design is about. Rather, they kinda discount it as »artsy«. So while it’s very simple and looks very good @Xaviju I’m not convinced we should use it.

**Comment by gillisig** [comment]
Here is a simpler version of the pencil. 
![osd](https://cloud.githubusercontent.com/assets/5390864/8082526/68a2839a-0f6d-11e5-8fb5-5277687724a2.png)

If you look closely it kinda looks like pacman eating one of those monsters. :smile:

**Comment by MariekevC** [comment]
I liked the idea of @jdittrich where the logo is 'in the making', @Xaviju brush is strong, but I guess miss the 'code' or technical side of design. 
All above examples gave this idea:
![osd_logo_openbrush](https://cloud.githubusercontent.com/assets/6065633/8082430/966327fc-0f7d-11e5-8883-75a0d9a480fb.png)

It seems I can't let go of the the open source logo, so I took @jdittrich idea of the unfinished one, added the brush voor the creative part. It can also be a " / ". Or together with the left part as " < / ".

**Comment by aoloe** [comment]
i love xaviju's logo, but there is an issue that bothers me, on that has been bothering me for a longer time for another logo: the scribus one

![scribus logo](https://upload.wikimedia.org/wikipedia/commons/thumb/8/85/Scribus_logo.svg/60px-Scribus_logo.svg.png)

and since the meaning of the proposed logos is questioned, as a non-designer, i dare to wonder why you are using tools for drawing on paper as a symbol for digital design.
the time has not come yet, when the digital world is real enough?

**Comment by plastelina** [comment]
hahahah, it does :) @gillisig! 

I've got a question: why did we stuck on b&w version of logo only? Design is all about colour, balance, simplicity ( I like @una direction with colour). Maybe it would be worth considering (as someone mentioned before) the direction for logo as open source hardware has. 

Also, FINALLY! bravo @aoloe . As a designer I do use biro for sketches ;), sometimes pencil to come up with a concept. 

What do you guys think about these? (RGB or CMYK), also I was thinking of incorporating ruler or some sort of grid (mobile, website) to the actual open source logo that we are currently using? Any thoughts?
![option1](https://cloud.githubusercontent.com/assets/7548189/8082714/12b47042-0f77-11e5-9c78-9bc4fb7b9520.png)
![option2](https://cloud.githubusercontent.com/assets/7548189/8082711/12af99dc-0f77-11e5-8db8-a85b2f395c14.png)
![option3](https://cloud.githubusercontent.com/assets/7548189/8082712/12b09a3a-0f77-11e5-93e0-6d62845115f7.png)
![option4](https://cloud.githubusercontent.com/assets/7548189/8082713/12b1cfcc-0f77-11e5-957a-127e9c14998d.png)





What do we th

**Comment by Xaviju** [comment]
Well, I don't think the pencil or the brush are trying to be 'real tools', but just social conventions. It fits well an 'open' circle + a social-convention-designer-tool to say 'Open Design'. 
Is it Brush | pencil = design? Yes and no.

@gillisig :____)
@aoloe agree with that reflection, I'll think about it, but its hard to make that abstraction 'real'
@plastelina Love it, specially option 2. Still miss the brush\pencil\pen\ruler\etc but its nice :)

We should be careful not to get stuck in 'what a designer is' because that is not a logo, that is probably a long book full of philosophical questions. :)

**Comment by jancborchardt** [comment]
@plastelina :D I just played around with a very similar concept:
![opensourcedesign-logo](https://cloud.githubusercontent.com/assets/925062/8083530/9297b0ae-0f85-11e5-8e96-9145c76a55b6.png)

**Comment by jancborchardt** [comment]
That is – I think this refers more nicely to what design is about: Color, shape, layout. I’m also thinking of exploring the »sketching« idea of @jdittrich more.

**Comment by plastelina** [comment]
LOVE IT! this is super cool.well done @Xaviju !!!
 Why are you guys so obsessed with pink and gradients ;), solid colours are easier to print on merchandise and look more professional, but on the other hand designers and developers of open source world are like http://1.bp.blogspot.com/-QBHor1ih_Bc/UcHiMyxmjfI/AAAAAAAAAEw/hSrU64vKNrk/s1600/cat-riding-a-fire-breathing-unicorn-16414-1680x1050.jpg :D

**Comment by pfefferle** [comment]
@jancborchardt I agree, but the logo should still work in b/w to use it for example in an icon font...

**Comment by jancborchardt** [comment]
Yeah I know, it should be simpler to work on an icon font and generally for monochrome. But that doesn’t mean we have to super-simplify the default logo. See Firefox for example, or Travis CI. We can always do a monochrome version of it, I just didn’t do it yet.

**Comment by simonv3** [comment]
Hey all!

Thanks so much for contributing all of your great thoughts, ideas, logos, sketches, etc. This is super awesome, and *exactly* what I hope for this community. It's great to see you all pitch in, and I'm super appreciative of all of the work you've been doing on this.

However, to prevent this from going on and on with new ideas and concepts, and never end up with a logo, I'm going to suggest that we all agree that a week from today Wednesday the 17th (this gives people a weekend to pitch in ideas as well) we close this topic. If by the end of this week we don't have a clear choice of favorite, we can put things to a vote.

I want *everyone's* opinions on *our* logo. Why a particular concept is great, why it fails, etc. I want to make sure that everyone's voice gets a chance to be heard. But I also want a positive conclusion to this conversation.

Let me know if you think this is unfair. I'll copy this to the first issue in this thread as well.

**Comment by una** [comment]
Great point @simonv3 :) We can then have a simple vote after that date

**Comment by plastelina** [comment]
fair enough @simonv3, I can outline a basic guideline for creating the logo this evening, i.e. purpose, colour, shape, etc. like this one http://www.smashingmagazine.com/2010/07/21/designing-style-guidelines-for-brands-and-websites/?

**Comment by una** [comment]
stemming off @plastelina, I love her ideas with the RGB filters:

![od-ideas2](https://cloud.githubusercontent.com/assets/1693164/8087767/5fb96826-0f62-11e5-8976-15c449ce460c.png)

**Comment by pfefferle** [comment]
@una @jancborchardt @plastelina I am not sure if it is a good idea to use the opensource logo in that way: http://opensource.org/logo-usage-guidelines

The pencil and brush are abstract enough I think...

**Comment by una** [comment]
i'm not opposed to changing out the center icon at all

**Comment by GarthDB** [comment]
Agreed.

**Comment by plastelina** [comment]
@pfefferle aren't you missing the point? please see the example- http://upload.wikimedia.org/wikipedia/commons/thumb/f/fd/Open-source-hardware-logo.svg/2000px-Open-source-hardware-logo.svg.png I'm not saying - use open source logo and add things (read elements), use the concept - open door lock. We aren't trying to copy or change the open source logo, we are trying to come up with one that is in line with open source community, at least that was my understanding. The whole point of this thread is to come up with unique logo, not copy open source logo. I think that the fact that it's been used on opensourcedesign.net as representative logo gives people misleading impression that it's a starting point for the ideas. Maybe it's time to think out of this circle (logo) with a lock and head this way? http://ibrandstudio.com/wp-content/uploads/2014/06/logo-design-trends-1.jpg
http://imgs.abduzeedo.com/files/articles/logo-design-polygons/ink_colourful_1x.jpg
http://a2.files.theultralinx.com/image/upload/c_fit,cs_srgb,dpr_1.0,q_80,w_620/MTI5MDI0ODc5ODA1OTYxNDkw.jpg
http://www.downgraf.com/wp-content/uploads/2014/03/Geometric-and-Polygon-Logo-Designs-33.png
OSD can be beautifully incorporated into the above...
Will read more about it and come back to you in regards to guidelines...

need to run, speak in a bit.

**Comment by MariekevC** [comment]
@plastelina A basic guideline would be good :-)

Below I put some things down wich I am wondering about. Maybe they can be of use.

* 'Colours'. The website http://opensourcedesign.net is monochrome with a splash of blue (for links). Same for the mock-ups for development of this site. 
  - A multi-coloured logo will centainly stand out. Is that favoured?
  - If multiple colours, then maybe they can respresent different sides of OSD (design, development, projects, etc.)?
  - Coloured and b/w version important?
  - Will it be used in an icon font? Is that a goal?
* 'Logo talks'. How can we avoid a logo that could reference to a place where design is presented (instead of a 'design/development hub')?
* Text yes/no?
  - Can the logo contain text/letters?
  - Will the text 'open source design' be added to the logo?

 (cc @simonv3 @jancborchardt )

**Comment by simonv3** [comment]
@MariekevC Just want to add that the colors of the current site are up for grabs. The current site's design is not final, and that includes the color scheme. The wireframes I've posted are just that - wireframes, not visually finished renditions. They're black and white because the focus is on the structure, not the visual. I'm personally open for experimenting with color.

**Comment by jancborchardt** [comment]
The site itself is basically style-wise only a wireframe. I’m cool with keeping that aesthetic though. The logo can be different.

Regarding text: Text should always only be optional of course, as in the _logo_ should not be a wordmark. But it’s also necessary that it looks good with »Open Source Design« added.

**Comment by plastelina** [comment]
Sorry for disappearing (work deadlines).

( cc @jancborchardt @simonv3 @una @MariekevC @Xaviju @pfefferle @GarthDB @teufelabgott , sorry if I've missed someone)

Here is the link ( http://karolinaferenz.com/logo_guidelines.pdf) to the logo checklist that I use before starting any conversation with a client :) / here: contributors. Obviously half of this list doesn't even apply, but it might be a good starting point.
If you guys could run through these points and sift through whatever is most relevant that would be fantastic. :) . I can't login to IRC to discuss this with more people...

I've got similar list for branding that could help with developing / styling this slick wireframe. (@jancborchardt) . Personally, I like the page layout, it's clean and simple to navigate but needs some styling love, as you've mentioned @jancborchardt, it's work in progress.

Also, I would like to post this job/task to behance job board. That could give OSD great exposure and maybe attract the right crowd, so it's not just echo chambers feedback :), what do you think?

Would it be possible to create some sort of shout out or mention on the main page (http://opensourcedesign.net/), i.e.: "1 week, 1 logo - design logo for OSD", that would lead to this thread? I have to check my email and then look for logo thread to view it in github, I know it's funny, but I'm still learning how to use this monster that is pretending to be a friendly kitty squid :) 

Thank you.

**Comment by jancborchardt** [comment]
I don’t want to be a naysayer – but in general logo contests leave a very sour taste in my mouth. That’s not necessarily the ethic I think we should push.

Because logo contests like this are basically used to get a bunch of logos to chose from, quickly, for cheap. And the result is nearly always not good. Sure it’s open source where everyone can contribute, but within our Open Source Design movement that still means we have to uphold the values of good design.

**Comment by plastelina** [comment]
@jancborchardt ok, so what do you suggest to do? 
I didn't have contest in my mind, from my perspective it looks like this is an echo chamber design. Would you mind explaining how do you want to uphold the values of good design without involving designers? (not attacking here, just asking). I believe there are a lot of good people out there who would love to get involved but don't know how and this is one of the ways for non technical people to jump in into open source, no?

**Comment by simonv3** [comment]
I'm all for putting out the word out there, and behance is a pretty good place to talk to designers I hear. 

I'll add a link to the website. If we want to go down the route doing this properly design wise with spec and features etc, which is probably smart, then maybe it makes sense to extend the deadline a bit?

**Comment by mrondina** [comment]
As I've been thinking about the design of the Open Design Foundation logo, there is an opportunity to use the diversity of the design community (print, web, interactive, etc) to build a design system. There can, of course, be a standard default  logo. If the circular element of the logo is carried forward, we can open the "O" to be replaced using logo marks. This would enable the redesign of the logo for many uses and opens it to the editorial of the Open Design community.

This is very much influenced by Paula Scher's work on the Philadelphia Museum of Art - http://www.underconsideration.com/brandnew/archives/new_logo_and_identity_for_philadelphia_museum_of_art_by_pentagram.php#.VX98qBNViko

I took some screenshots of the logo work in this thread and made a couple of quick sketch-up of the idea:

![design-open-system-1](https://cloud.githubusercontent.com/assets/938691/8174114/4eb1b270-13a6-11e5-9ffe-99d08dccedf0.jpg)
![design-open-system-2](https://cloud.githubusercontent.com/assets/938691/8174118/5ba3b488-13a6-11e5-95d8-3fd0edeca2a1.jpg)

**Comment by jancborchardt** [comment]
@mrondina just as a sidenote: That can very easily be confused with »Pen Design« – especially since the logo in the first case is a pen. ;)

**Comment by MariekevC** [comment]
@mrondina 's input is a very nice idea. The examples have an open friendly feeling. 
Maybe go a bit more abstract on the sign (i.e. deconstruct the open source logo) and just add the text to it, like so:
![osd_logo_open](https://cloud.githubusercontent.com/assets/6065633/8189940/e289525a-145f-11e5-8436-09fbdf4a03c6.png)

**Comment by bnvk** [comment]
Heya everyone. Sorry i've been absent,
Firstly, THANK YOU, all the ideas in this thread are awesome. Personally, my favorite is the color renderings @una did of @GarthDB treatment of @jdittrich "pencil" idea. It is absolutely wonderful, sharp, and vivid and something I think would really be great to use as is. I also really like the RGB "overlay" rendering @jancborchardt did.

@Xaviju could you post a vector of the brush rendering you did somewhere? :smile: 

**Recruiting outside designers (Behance, etc)** I don't think we should look outside of current community for that. Maybe a couple tweets about this thread... but between this, the IRC room, and members of this github org who follow along!

**Picking a logo** are we still planning on voting on Wed the 17th?  I've been laid up sick in bed and then in the hospital, just getting back on my feet. Could we do the 18th or 19th? I'd really like to try to do a few renderings combining what I like best of all of these things to contribute to the thread / voting options!

I think we're close to having an awesome logo! You all are great :grin:

**Comment by GarthDB** [comment]
@bnvk what is the process for voting?

**Comment by bnvk** [comment]
@GarthDB hehe, I'm not too sure. The simplest idea i'm thinking of are a survey monkey / google form with:

- Each distinct logo variation 
- A rating of 1 - 10 on each option

This would yield a score of points that should give us some idea of what the community likes most / least.

There's also a little voting tool Mailpile made called HoiPoi for our community that allows an invited by email address only simple "ordering of your least -> most favorite" and builds the options out of Github Issues.

**Comment by simonv3** [comment]
@bnvk @GarthDB and everyone else: 

I've created an issue to discuss this specifically https://github.com/opensourcedesign/organization/issues/10.

**Comment by Xaviju** [comment]
@bnvk sure, the brush logo SVG vector > https://www.dropbox.com/s/xapnj5yhaihtw2x/opendesign-sketch.png?dl=0 :tada:

**Comment by jmarlena** [comment]
Here's an idea for feedback/input or to inspire other iterations that displays binary. 

![opendesign_logo](https://cloud.githubusercontent.com/assets/6732600/8220924/77d5261a-150d-11e5-8e44-45d52540083e.png)
![opendesign_logo2](https://cloud.githubusercontent.com/assets/6732600/8221138/6f903cc2-150f-11e5-9482-98f36ec7dd9f.png)

**Comment by GarthDB** [comment]
Just to help keep this tangled web we weave straight, it's Open Source Design.

Thanks for contributing @jmarlena

**Comment by bnvk** [comment]
Here are my two tiny remixes from my favorite bits and pieces you fine folks created thus far

![bnvk-logo-1](https://cloud.githubusercontent.com/assets/48677/8252354/bc368d22-1674-11e5-805d-a33aebf3b76b.png)

Both renderings could be reduced down to the "pencil" and the "brush" as a monochrome logo, which so far are quite strong IMO!

**Comment by una** [comment]
<3 that 2nd one @bnvk

**Comment by jancborchardt** [comment]
The primary colors are a bit too garish, no? Here’s my remix with @Xaviju's brush icon:
![opensourcedesign-logo-brush-demo](https://cloud.githubusercontent.com/assets/925062/8259037/db224600-16b9-11e5-880c-fb505c3d07f3.png)
([Source SVG with all steps](https://cloud.jancborchardt.net/public.php?service=files&t=983283b5c6b6c1c12b5a34ebd10aa22b))

**Comment by jdittrich** [comment]
@una since I liked your pencil-based logo ideas very much, I’d like to build upon them (if that’s o.k. with you). Could you share the source files?

**Comment by una** [comment]
yep! here's a link: https://www.dropbox.com/s/q80b3vyb47u2kdi/opensrcdesign_02.svg?dl=0 

For some reason I couldn't find the original file I was working on :( with all of the above logos. I will keep looking

**Comment by GarthDB** [comment]
@jdittrich @una The building upon is my favorite part.

**Comment by jdittrich** [comment]
I’d like to throw this one in: 
![bildschirmfoto vom 2015-06-23 13 14 57](https://cloud.githubusercontent.com/assets/3416487/8304862/f59603da-19a9-11e5-8bcf-c4b332ebd324.png)
It is very much @una’s logo (1st row, 2nd column), except that the circle is darker (equivalent to the pencils tip; more contrast to be better visible). And since many designers still rely on pencils for various reasons I still like it as a symbol of our craft.

**Comment by MariekevC** [comment]
A couple of remixes to add. 
![osd_logo_openkleur](https://cloud.githubusercontent.com/assets/6065633/8315122/9355d598-19ee-11e5-95be-6f0db40ff080.png)
![osd_logo_brushopen](https://cloud.githubusercontent.com/assets/6065633/8315125/9777550c-19ee-11e5-8363-f807e13ca3af.png)
The first one is the one I made earlier. Abstract detail of opensource logo and 'brush' but updated to sharper lines and a splash of gradient color I liked in @jancborchardt logo. The second is based on the first, but replaced the abstract part with @Xaviju brush.

**Comment by MariekevC** [comment]
Maybe I should add this one also:
![osd_logo_opensourcedesign](https://cloud.githubusercontent.com/assets/6065633/8315530/bce08892-19f1-11e5-9326-1998f9ecfdd4.png)
Taking @jancborchardt logo apart to check the gradients, this happened. A bit quirky maybe, but the colors attract and then there are 2 opensource logo's (Design/Development), combined in the middle by a third one: "Bringing stuff together for opensource". It could also easily be animated into Jan's orignal (for presentation or someting).

( [And a combined svg for the last designs](https://owncloud.thisnet.nl/index.php/s/f1KTAKM6AhQ0KrX) )

**Comment by una** [comment]
This has all been some great collab -- When are we voting/deciding?

**Comment by simonv3** [comment]
I agree, this has been some awesome collab. Like I mentioned [earlier](https://github.com/opensourcedesign/opensourcedesign.github.io/issues/18#issuecomment-112594376), we've started discussing how we'll [reach decisions like this in another thread](https://github.com/opensourcedesign/organization/issues/10).

The last few comments in that thread seem to indicate we'll have a call to which everyone is invited on Sunday the 5th of July. Pitch in if that works / doesn't work for you. Pitch in if you have thoughts about how to reach consensus, etc.

**Comment by Bojhan** [comment]
Just wondering from a branding standpoint. What do we want this logo to conceive? Is there a certain "feeling" we are aiming for?

**Comment by GarthDB** [comment]
@bojhan what feelings do you think it should have?

**Comment by simonv3** [comment]
Just realized I hadn't posted this here yet, but every logo in this thread now automatically gets added to this page on the website: http://opensourcedesign.net/logos. I tried to do the best I could with sizing etc.

**Comment by plastelina** [comment]
@simonv3 :+1: good stuff! :)

**Comment by gillisig** [comment]
Well I decided to give it a go and made this soft cornered version of the open source icon. 

<img width="236" alt="osd-logo" src="https://cloud.githubusercontent.com/assets/5390864/8628027/c12f80c4-273e-11e5-9818-ff7dc229b02f.png">

**Comment by nicknikolov** [comment]
This is the best one I've seen yet @gillisig, good work!

**Comment by jancborchardt** [comment]
@gillisig yeah … simple and it rocks!

**Comment by una** [comment]
I like the typeface a lot on that last one by @gillisig :)

**Comment by MariekevC** [comment]
@gillsig Yes, like the type face. The color does a neat trick to my eyes when scrolling (purplish outlines). 
For the icon itself, the form, does this resemble a sausage (Wurst)? No ill meaning. It could be a hilarious side effect, if that is what we want.

**Comment by elioqoshi** [comment]
I love @Xaviju 's version. It's simple and elegant. I strongly believe we should go with that :)

**Comment by jancborchardt** [comment]
I’m really particular about us not using the brush. It puts too much emphasis on »art«.

My favorites are:
* [the pen](https://github.com/opensourcedesign/opensourcedesign.github.io/issues/18#issuecomment-110488602) by @jdittrich, @GarthDB & @gillisig 
* [the color spectrum](https://github.com/opensourcedesign/opensourcedesign.github.io/issues/18#issuecomment-110751788) by @plastelina & me
* not sure about [the »sausage«](https://github.com/opensourcedesign/opensourcedesign.github.io/issues/18#issuecomment-120509338) by @gillisig ;) Maybe it works better in black

If anyone is up for another proposal, maybe we should play around more with a logo on the topic of sketching / mockups / planning.

**Comment by una** [comment]
Maybe we should set a deadline on making a decision? And give everybody 1-2 votes?

**Comment by jancborchardt** [comment]
Well, we don’t need to find a perfect logo in the first iteration. It would be good if people interested/invested in this could just state what they lean to, as I did above. Then we can see maybe which direction we want to explore more.

**Comment by elioqoshi** [comment]
The brush is not an icon which limits itself to art, I wouldn't see that as an argument to not use a brush in the logo. @jancborchardt

**Comment by simonv3** [comment]
Okay! I think it's past time to take a survey and see how people feel. Anyone have any tools they recommend?

**Comment by plastelina** [comment]
I personaly feel that none of the logos is strong enough to represent open source design, can we just hold on with this one...promote fosdem more and maybe suggest logo competition at fosdem this year? I don't think there is a clear mission, message behind current designs...the fact that we are using art tools in a circle is quite qimmicky, literal and weak. Can we wait with the actual logo and maybe attack the subject around November?

**Comment by GarthDB** [comment]
Wouldn't iterative design be a great way to represent open source design? Pick a logo, and make improvements as we go. If we wait for the perfect logo we could be logoless for a long time.

What steps do you think we should take @plastelina to avoid a weak logo? We could still carry the conversation while preparing for Fosdem; what process would help us the best here?

**Comment by nicknikolov** [comment]
One of the options in the vote could be "None of these are good enough"

**Comment by plastelina** [comment]
I think we have amazing start already, there are a lot of great propositions @GarthDB . I'm not a logo designer, but there are great logo designer already in this group. As a designer I do struggle with a middle management problem and I think that is what we are hitting here - to much talking not much doing and mostly, diluting designs and leaving us all with too many choices. When I work with a client I normally suggest two option - never more, otherwise job won't be done for ages...

 My suggestion would be to write a brief ( I've already shared a logo design checklist), so ask yourself ( as in all of us) : what would you like this logo to represent and what message would you like to send? How close does it have to mimic the opensource existing logo or at all? and keep it simple (maybe just open source design in good font would be enough for the start). (  I've got a feeling that somewhere in these posts there is a collection of thoughts on this topic so maybe someone faster then me with great searching/summary skills could pick it up and wrap it in one post and he have brief ready - voilà! :) )

I've done some research (please don't take it personally) and @una could be the person to lead the logo design, she is very good at this and have some amazing ideas. Her logos are very creative and fun, also simple and most importantly spot on briefs. Maybe if @una would agree and everyone else would chip in to create a brief we could achieve the goal sooner :).  yay/nay?

Fun Fact
On the other hand, some famous logos took 100 years to develop, so we've got plenty of time :smile_cat:  :dancer: , i.e., http://www.hongkiat.com/blog/logo-evolution/

cc: @simonv3 @jancborchardt @MariekevC @gillisig 

Also,
I would like to design a poster for FOSDEM 2015 dev/design room this year and post it here so everyone can print it and share it in their home town/uni/work maybe twitter or keep it :), also if we will manage to establish a logo I'm happy to get it ready for print for t-shirts, badges, pencils, stickers,etc. if that's the case.

**Comment by GarthDB** [comment]
@plastelina I'm a big fan of having a creative director even if it's on a project by project basis. @una is an excellent choice if she has the time/energy.

The stuck in committee problem is solved by having an owner. That doesn't mean other people shouldn't provide input but ultimately someone has to make a decision; even if that decision is to vote (not my favorite decision strategy but better than indecision).

**Comment by simonv3** [comment]
Agreed with both @plastelina and @GarthDB.

Here's my answers to @plastelina's questions:

>  What would you like this logo to represent and what message would you like to send?

Open Source Design is a **community** of designers and developers who want to **push design** in Open Source further.

> How close does it have to mimic the opensource existing logo or at all?

I'm not too fussed about this - but I do like the neat connection. On the other hand, I'm wary that we're being too liberal with the Open Source design logo - it has a variety of [restrictions](http://opensource.org/logo-usage-guidelines). If someone comes up with something completely new that fits, I'd be all for that.

> Are you open to evolution of the logo? *(I added this myself)*
Yes!

**Comment by razetime** [comment]
How about we keep the logo simple?
Here's something I made:
![path4136](https://cloud.githubusercontent.com/assets/11015319/10023275/e01d366a-616e-11e5-8fd2-afd45a70519f.jpg)

**Comment by una** [comment]
@razetime it seems a little dark :/ especially think about using it in various places

**Comment by razetime** [comment]
@una Yes, it is rark, but we can lighten it for use in different areas.

**Comment by bnvk** [comment]
@razetime it's so dark I can barely see the definition of the open source icon. I think the basic criteria we are aiming for are both:

1. Contrasted enough that it can be a single color on any color background + be discernible
2. Expressive of "open source" as well as something "design" related

Many of the treatments above (mine included) fail at the first criteria by themselves, since they are elaborate + multi-colored, but they are building on top of monochrome foundations of others. It's important to keep that in mind :)

**Comment by simonv3** [comment]
Hey all!

So, that's enough logo options I think. As a group of volunteers, I don't think anyone has a significant amount of time to take a lead on logo design, or taking the time to appoint someone to do so. After talking with some people on the [IRC channel](http://chat.opensourcedesign.net/) I put up a survey asking people to indicate their favorite logo.

## You can [take the survey here](https://open-source-design-logo.herokuapp.com/).

You can enter your favorite choice and your second favorite choice. This will be a simplified [single transferable vote](https://en.wikipedia.org/wiki/Single_transferable_vote) survey. That means that if your first choice comes last in the first round (so it gets eliminated), your second choice vote will be counted as well, etc until there's a clear winner. The results are stored anonymously, so I'll publish them once they're done (unless people have a major objection?).

If you have a better idea, please tell us. There's a special comment box. It can be anonymous, but I'd prefer for it to be actionable, and that you are willing to take the lead on this. Your idea might be great and the best way to do it, but if no one acts on it than it won't happen. (See: this thread).

I'm going to close this topic to prevent more options from appearing. If you think this is wrong, ping me on IRC or on twitter (@simonv3 on both), or any of the other people who can open it again. Also you can open another issue for discussion purposes specifically. 

(The reason this took so long is because I couldn't find a suitable open source survey tool. If you want to help me build this one out more, [come check it out](https://github.com/simonv3/quick-survey/) ;))

**Comment by simonv3** [comment]
Hey all! Just a ping to all involved. I'll be closing the survey in 3 days (it will have been open for two weeks). Someone's reached out to offer to lead a real design charge on this (I'll be in touch). I'll publish the results and a link to the anonymized CSV file. Then I'll discuss with the person who stepped forward whether they'd be okay with using the winner as an interim logo.

If you want to share around the survey one final time, now's your chance.

I'll create a new issue with the results and what I propose to do. Also, please check out the bylaws issue: https://github.com/opensourcedesign/organization/issues/12

**Comment by GarthDB** [comment]
@simonv3 that's great news.

**Comment by jancborchardt** [comment]
@simonv3 good stuff! Unfortunately I was on vacation exactly those 2 weeks and missed the survey. But I guess enough people voted. :)

**Comment by simonv3** [comment]
@jancborchardt Bummer! Sorry to hear that. 28 people voted - which is a good amount, but I kind of wish it had been more. See results in #28.
